### PR TITLE
feat: add Claude (Anthropic) conversation import support

### PR DESCRIPTION
**Pull request by Pleasurecruise**
### What this PR does

**Before this PR:**
Cherry Studio had no support for importing Anthropic/Claude conversation exports. Users had no way to migrate their Claude chat history into Cherry Studio.

**After this PR:**
- Adds `AnthropicImporter` to parse Claude's official `conversations.json` export format, converting messages, content blocks, and topics into Cherry Studio's internal data model, and mapping assistant messages to the corresponding Claude model identifier.
- Registers `AnthropicImporter` in `importers/index` so it is wired into the unified import pipeline.
- Adds `AnthropicImportPopup` modal component with file selection, progress feedback, and error display.
- Hooks the entry point into `ImportMenuSettings` so users can trigger a Claude export import directly from the settings UI.
- Adds i18n strings for `en`, `zh-CN`, and `zh-TW`, covering all UI copy and error messages.

Fixes #

### Why we need it and why it was done in this way

Many users run Claude and Cherry Studio side by side, and migrating conversation history is a common need. This implementation follows the existing importer conventions (modeled after the OpenAI/ChatGPT importer) to keep the architecture consistent:

- **Parser and UI are decoupled**: `AnthropicImporter` handles only data transformation; the popup component handles only interaction. Each can be maintained and tested independently.
- **Model mapping**: Claude exports include a `model` field on assistant messages. Mapping it directly preserves the user's model context rather than collapsing everything to a default.
- **Lightweight validation first**: Only the top-level structure of `conversations.json` is validated, so minor format changes in future Anthropic exports do not break the entire import.

### Breaking changes

None. This PR is purely additive. No existing Redux store structures, data models, or IndexedDB schemas are modified.

### Special notes for your reviewer

1. Please pay close attention to whether the content block type enumeration in `AnthropicImporter` fully covers the current Claude export format (`text`, `tool_use`, `tool_result`, etc.).

### Checklist

- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: Write code that humans can understand and Keep it simple
- [x] Refactor: You have left the code cleaner than you found it (Boy Scout Rule)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Documentation: A user-guide update on docs.cherry-ai.com covering "Import Claude conversations" would be useful — can be submitted as a follow-up PR
- [x] Self-review: I have reviewed my own code before requesting review from others

### Release note

```release-note
Added Anthropic/Claude conversation import: users can now go to Settings → Data → Import, select a conversations.json file exported from Claude, and import their chat history into Cherry Studio. Original model information is preserved, with full UI support for English, Simplified Chinese, and Traditional Chinese.
```

### CODE REVIEW

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+      role,
+      assistantId,
+      topicId,
+      createdAt,
+      updatedAt: anthropicMessage.updated_at ?? createdAt,
+      status: role === 'user' ? UserMessageStatus.SUCCESS : AssistantMessageStatus.SUCCESS,
+      blocks: [blockId],
+      // Set model for assistant messages to display Claude logo
+      ...(role === 'assistant' && {
+        model: {
```
Comment (GeorgeDong32) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:338]: > [!NOTE]
> This comment was translated by Claude.

**🔴 Model Mapping Doesn't Match PR Description**

The PR description mentions:
> "Mapping assistant messages to the corresponding Claude model identifier"

However, the actual code hardcodes all assistant messages as `claude-sonnet-4-6`, instead of reading the actual model information from the exported data.

**Suggestions**:
1. If the Anthropic export format includes a model field, it should be read from there
2. If not, I recommend updating the PR description to explain the current approach (hardcoding the default model)

---
<details>
<summary>Original Content</summary>

**🔴 模型映射与 PR 描述不符**

PR 描述中提到：
> "Mapping assistant messages to the corresponding Claude model identifier"

但实际代码将所有 assistant 消息硬编码为 `claude-sonnet-4-6`，而不是从导出数据中读取实际模型信息。

**建议**：
1. 如果 Anthropic 导出格式包含模型字段，应该从中读取
2. 如果不包含，建议更新 PR 描述说明当前行为（硬编码默认模型）

</details>

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+const logger = loggerService.withContext('AnthropicImporter')
+
+/**
+ * Anthropic Claude Export Format Types
+ */
+interface AnthropicCitation {
+  uuid: string
+  start_index: number
+  end_index: number
+  details: {
```
Comment (GeorgeDong32) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:27]: > [!NOTE]
> This issue/comment/review was translated by Claude.

**🟡 Unused Type Definitions**

The `AnthropicCitation` and `AnthropicAttachment` interfaces are defined but not used in the code.

**Suggestions**:
1. Remove unused interfaces to keep code clean
2. Or add comments explaining why they are retained (e.g., reserved for future features)

---
<details>
<summary>Original Content</summary>

**🟡 未使用的类型定义**

`AnthropicCitation` 和 `AnthropicAttachment` 接口已定义但未在代码中使用。

**建议**：
1. 删除未使用的接口以保持代码整洁
2. 或者添加注释说明保留原因（如为未来功能预留）

</details>

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          })
+        )
+        setOpen(false)
+      } else {
+        window.toast.error(result.error || t('import.claude.error.unknown'))
+      }
+    } catch {
+      window.toast.error(t('import.claude.error.unknown'))
+      setOpen(false)
+    } finally {
```
Comment (GeorgeDong32) [inline src/renderer/src/components/Popups/AnthropicImportPopup.tsx:55]: > [!NOTE]
> This issue/comment/review was translated by Claude.

**🟡 Error Handling Loses Context**

The catch block does not capture the error object, losing debugging information:

```typescript
// Current
} catch {
  window.toast.error(t('import.claude.error.unknown'))
  setOpen(false)
}

// Suggested
} catch (error) {
  logger.error('Import failed:', error as Error)
  window.toast.error(t('import.claude.error.unknown'))
  // Consider not auto-closing the modal to allow users to retry
}
```

---
<details>
<summary>Original Content</summary>

**🟡 错误处理丢失上下文**

catch 块未捕获错误对象，丢失了调试信息：

```typescript
// 当前
} catch {
  window.toast.error(t('import.claude.error.unknown'))
  setOpen(false)
}

// 建议
} catch (error) {
  logger.error('Import failed:', error as Error)
  window.toast.error(t('import.claude.error.unknown'))
  // 考虑不自动关闭 modal，让用户可以重试
}
```

</details>

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+  const onCancel = () => {
+    setOpen(false)
+  }
+
+  const onClose = () => {
+    resolve({})
+  }
+
+  AnthropicImportPopup.hide = onCancel
```
Comment (GeorgeDong32) [inline src/renderer/src/components/Popups/AnthropicImportPopup.tsx:69]: > [!NOTE]
> This comment was translated by Claude.

**🟡 Component Side Effects**

Modifying the static property `AnthropicImportPopup.hide = onCancel` during component rendering is not recommended.

This should be handled in `useEffect` or by refactoring the component structure.

---
<details>
<summary>Original Content</summary>

**🟡 组件内副作用**

在组件渲染期间修改静态属性 `AnthropicImportPopup.hide = onCancel` 是不推荐的做法。

这应该在 `useEffect` 中处理或重构组件结构。

</details>

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+      {selecting && (
+        <div style={{ textAlign: 'center', padding: '40px 0' }}>
+          <Spin size="large" />
+          <div style={{ marginTop: 16 }}>{t('import.claude.selecting')}</div>
+        </div>
+      )}
+      {importing && (
+        <div style={{ textAlign: 'center', padding: '20px 0' }}>
+          <Progress percent={100} status="active" strokeColor="var(--color-primary)" showInfo={false} />
+          <div style={{ marginTop: 16 }}>{t('import.claude.importing')}</div>
```
Comment (GeorgeDong32) [inline src/renderer/src/components/Popups/AnthropicImportPopup.tsx:111]: > [!NOTE]
> This issue/comment/review was translated by Claude.

**🟢 Fake Progress Bar**

The progress bar always shows 100%, with no actual progress tracking.

**Suggestions**:
1. Implement real progress tracking (based on the number of messages processed)
2. Or simplify to an indeterminate loading animation (remove the Progress component, keep only Spin)

---
<details>
<summary>Original Content</summary>

**🟢 虚假进度条**

进度条始终显示 100%，没有实际进度跟踪。

**建议**：
1. 实现真正的进度跟踪（基于已处理的消息数）
2. 或者简化为不确定进度的加载动画（移除 Progress 组件，仅保留 Spin）

</details>

**Review by GeorgeDong32 — COMMENTED** [review]
> [!NOTE]
> This review was translated by Claude.

## Code Review Summary

Thank you for your contribution! This PR adds Anthropic/Claude conversation import functionality. The overall implementation follows the existing importer pattern with a well-designed architecture.

### 📊 Review Results

| Category | Count |
|----------|-------|
| 🔴 Critical Issues | 1 |
| 🟡 Moderate Issues | 3 |
| 🟢 Minor Issues | 1 |

### 🔴 Critical Issues

1. **Model mapping does not match PR description** - The PR claims it will map to corresponding model identifiers, but the actual code hardcodes to `claude-sonnet-4-6`

### 🟡 Moderate Issues

1. **Unused type definitions** - `AnthropicCitation` and `AnthropicAttachment` interfaces are defined but not used
2. **Error handling loses context** - catch blocks do not capture error objects
3. **Side effects within components** - Modifying static properties during rendering

### 🟢 Minor Issues

1. **False progress bar** - Progress bar always shows 100%

### ✅ Strengths

- Architecture maintains consistency with existing ChatGPTImporter
- Complete internationalization (en/zh-CN/zh-TW)
- Clear code structure with good separation of concerns
- No security risks

### 📝 Recommendations

It is recommended to resolve the critical issue (model mapping or update PR description) before merging. The moderate issues can be addressed as follow-up optimizations.

---
<details>
<summary>Original Content</summary>

## 代码审查总结

感谢贡献！本 PR 添加了 Anthropic/Claude 对话导入功能，整体实现遵循现有的导入器模式，架构设计合理。

### 📊 审查结果

| 类别 | 数量 |
|------|------|
| 🔴 关键问题 | 1 |
| 🟡 中等问题 | 3 |
| 🟢 小问题 | 1 |

### 🔴 关键问题

1. **模型映射与 PR 描述不符** - PR 声称会映射到对应的模型标识符，但实际代码硬编码为 `claude-sonnet-4-6`

### 🟡 中等问题

1. **未使用的类型定义** - `AnthropicCitation` 和 `AnthropicAttachment` 接口已定义但未使用
2. **错误处理丢失上下文** - catch 块未捕获错误对象
3. **组件内副作用** - 在渲染期间修改静态属性

### 🟢 小问题

1. **虚假进度条** - 进度条始终显示 100%

### ✅ 优点

- 架构与现有 ChatGPTImporter 保持一致
- 国际化完整（en/zh-CN/zh-TW）
- 代码结构清晰，职责分离良好
- 无安全风险

### 📝 建议

建议在合并前解决关键问题（模型映射或更新 PR 描述），中等问题可作为后续优化。

</details>

File: src/renderer/src/services/import/importers/AnthropicImporter.ts (hunk 1 above)
Comment (Pleasurecruise) [reply src/renderer/src/services/import/importers/AnthropicImporter.ts:338]: > [!NOTE]
> This issue/comment/review was translated by Claude.

Anthropic's export format does not contain model information at all. This can be verified from conversations.json: each chat_messages only has uuid, text, content, sender, created_at, updated_at, attachments, files fields, and no model field.

---
<details>
<summary>Original Content</summary>

Anthropic 的导出格式根本不包含模型信息。可以从conversations.json中验证：每条 chat_messages 只有uuid、text、content、sender、created_at、updated_at、attachments、files 字段，没有任何 model 字段。

</details>

File: src/renderer/src/services/import/importers/AnthropicImporter.ts (hunk 2 above)
Comment (Pleasurecruise) [reply src/renderer/src/services/import/importers/AnthropicImporter.ts:27]: > [!NOTE]
> This issue/comment/review was translated by Claude.

Both AnthropicCitation and AnthropicAttachment are being used

- AnthropicCitation → Used in AnthropicContentBlock line 62: `citations?: AnthropicCitation[]`
- AnthropicAttachment → Used in AnthropicMessage line 80: `attachments?: AnthropicAttachment[]`

Added specifically to avoid using `any` and `unknown`

---
<details>
<summary>Original Content</summary>

AnthropicCitation 和 AnthropicAttachment 都有被使用
                                                                           
- AnthropicCitation → 用于 AnthropicContentBlock 第 62 行：citations?: AnthropicCitation[]                                                                      
- AnthropicAttachment → 用于 AnthropicMessage 第 80 行：attachments?: AnthropicAttachment[]

为了防止用 any 和 unknown 特意加上的

</details>

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx (hunk 5 above)
Comment (Pleasurecruise) [reply src/renderer/src/components/Popups/AnthropicImportPopup.tsx:111]: > [!NOTE]
> This issue/comment/review was translated by Claude.

Since Anthropic import is synchronous JSON file parsing (usually completed in milliseconds), implementing real per-message progress tracking would add unnecessary complexity. Normal AI code reviews don't have this.

---
<details>
<summary>Original Content</summary>

由于 Anthropic 导入是同步解析 JSON 文件（通常毫秒级完成），实现真实的逐消息进度追踪会增加不必要的复杂度 ai code review 的没一个正常的

</details>

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+      blocks: [blockId],
+      // Set model for assistant messages to display Claude logo
+      ...(role === 'assistant' && {
+        model: {
+          id: 'claude-sonnet-4-6',
+          provider: 'anthropic',
+          name: 'Claude Sonnet 4.6',
+          group: 'Claude 4.6'
+        }
+      })
```
Comment (GeorgeDong32) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:344]: **Issue: Hardcoded model information**

The model is hardcoded to `claude-sonnet-4-6` for all assistant messages. This doesn't reflect the actual model used in the original conversation.

**Suggestion**: Consider one of these approaches:
1. Extract the model from the message content blocks if available (Claude exports may include model info in the content)
2. Use a more generic identifier like `claude-3-opus`, `claude-3-sonnet`, etc. based on the conversation timestamp
3. At minimum, document this limitation in the code

Compare with ChatGPTImporter which uses `gpt-5` - both importers have this same issue of not preserving actual model information.

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 7]
```diff
@@ -0,0 +1,282 @@
...
+        .map((block) => block.text!.trim())
+
+      if (textParts.length > 0) {
+        return textParts.join('\n\n')
+      }
+    }
+
+    // Fallback to top-level text field
+    return message.text?.trim() ?? ''
+  }
```
Comment (GeorgeDong32) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:193]: **Issue: Only `text` blocks are processed, tool calls are silently dropped**

The `extractTextContent` method only extracts `type === 'text'` blocks. According to the type definitions (lines 56-71), Anthropic exports can contain:
- `tool_use` blocks (id, name, input)
- `tool_result` blocks (tool_use_id, content)

These are silently ignored, which means tool interactions in the original conversation are lost.

**Suggestion**: 
1. At minimum, add a warning log when non-text blocks are encountered
2. Consider representing tool calls in a simplified text format (e.g., `[Tool: {name}]` with input/result summary)
3. Or document this limitation in the UI/i18n description (currently says "only text, no images/attachments")

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+            messages: result.messagesCount
+          })
+        )
+        setOpen(false)
+      } else {
+        window.toast.error(result.error || t('import.claude.error.unknown'))
+      }
+    } catch {
+      window.toast.error(t('import.claude.error.unknown'))
+      setOpen(false)
```
Comment (GeorgeDong32) [inline src/renderer/src/components/Popups/AnthropicImportPopup.tsx:54]: **Issue: Error swallowed without logging**

The catch block catches the error but doesn't log it for debugging purposes. This makes troubleshooting user-reported issues difficult.

**Suggestion**: Add error logging before showing the toast:
```typescript
} catch (error) {
  logger.error('Claude import failed:', error as Error)
  window.toast.error(t('import.claude.error.unknown'))
}
```

Note: You'll need to import `loggerService` from `@logger` similar to the importer.

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 9]
```diff
@@ -0,0 +1,282 @@
...
+  url?: string
+  text?: string
+  is_missing?: boolean
+  metadata?: {
+    type: string
+    site_domain?: string
+    favicon_url?: string
+    site_name?: string
+  }
+}
```
Comment (GeorgeDong32) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:53]: **Note: Type definitions for future use**

These type definitions (`AnthropicCitation`, `AnthropicAttachment`, `AnthropicFile`, `AnthropicToolResultItem`) are defined but not currently used in the processing logic.

This is fine if they're intentionally defined for future enhancement (e.g., handling attachments, citations, or tool results). However, if they're not planned to be used, consider removing them to reduce code complexity.

If keeping for future use, consider adding a comment like:
```typescript
// Type definitions for future enhancement (attachments, citations, tool results)
```

**Review by GeorgeDong32 — COMMENTED** [review]
## Review Summary

This PR adds Anthropic/Claude conversation import support, following the existing ChatGPT importer pattern. The implementation is well-structured and follows project conventions.

### ✅ Strengths

1. **Good architecture** - Clean separation between parser (`AnthropicImporter`) and UI (`AnthropicImportPopup`)
2. **Follows existing patterns** - Consistent with `ChatGPTImporter` implementation
3. **Proper i18n** - All user-facing strings are internationalized
4. **Good validation** - Format detection distinguishes from ChatGPT format using `!('mapping' in conv)`

### 📝 Suggestions

1. **Missing unit tests** - The importer logic (282 lines) lacks test coverage. Consider adding tests for:
   - `validate()` method with various inputs
   - `parse()` method with sample conversations
   - Edge cases (empty conversations, missing fields)

2. **Hardcoded model** (line 213-220) - All assistant messages get `claude-sonnet-4-6` regardless of actual model used

3. **Tool calls ignored** - Only `text` blocks are processed; `tool_use`/`tool_result` blocks are silently dropped

4. **Error logging** - Catch block in popup doesn't log errors for debugging

### 🔍 Files Reviewed

| File | Status |
|------|--------|
| `AnthropicImporter.ts` | ✅ Reviewed |
| `AnthropicImportPopup.tsx` | ✅ Reviewed |
| `importers/index.ts` | ✅ LGTM |
| `ImportMenuSettings.tsx` | ✅ LGTM |
| i18n files | ✅ LGTM |

Overall, this is a solid addition. The suggestions above are improvements rather than blockers. Thank you for following the existing patterns and providing comprehensive i18n!

File: src/renderer/src/services/import/importers/AnthropicImporter.ts (hunk 6 above)
Comment (EurFelux) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:344]: **Issue: Model hardcoded to `claude-sonnet-4-6` for all assistant messages**

The PR description states "mapping assistant messages to the corresponding Claude model identifier", but the code hardcodes a single model for every assistant message. Claude's export JSON includes a `model` field on assistant messages (e.g. `claude-3-5-sonnet-20241022`, `claude-3-opus-20240229`). This should be extracted and mapped.

**Suggestion:**

1. Add `model?: string` to `AnthropicMessage` interface
2. Map the model string from the export to a Cherry Studio model object, or at least use it as the `id` field

```typescript
// Example approach
const modelId = anthropicMessage.model || 'claude-sonnet-4-6'
...(role === 'assistant' && {
  model: {
    id: modelId,
    provider: 'anthropic',
    name: modelId,
    group: 'Claude'
  }
})
```

If dynamic mapping is out of scope for this PR, please update the PR description to document this as a known limitation.

File: src/renderer/src/components/Popups/AnthropicImportPopup.tsx (hunk 3 above)
Comment (EurFelux) [inline src/renderer/src/components/Popups/AnthropicImportPopup.tsx:55]: **Issue: Error swallowed without logging**

The catch block discards the error object entirely, making it impossible to debug user-reported import failures from logs.

```diff
-    } catch {
-      window.toast.error(t('import.claude.error.unknown'))
+    } catch (error) {
+      loggerService.withContext('AnthropicImportPopup').error('Claude import failed:', error as Error)
+      window.toast.error(t('import.claude.error.unknown'))
```

File: src/renderer/src/services/import/importers/AnthropicImporter.ts (hunk 7 above)
Comment (EurFelux) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts]: **Issue: `tool_use` / `tool_result` blocks silently dropped**

`extractTextContent` only processes `type === 'text'` blocks. When a conversation contains tool calls (which is common with Claude), those messages may have _only_ `tool_use`/`tool_result` blocks and no `text` block — resulting in the entire message being filtered out by `convertConversationToTopic` (line 247: `text.length > 0`).

This means tool-heavy conversations could lose significant context or appear incomplete after import.

**Suggestion:** At minimum, log a warning when non-text blocks are skipped. Ideally, represent tool calls in a simplified text format:

```typescript
// e.g. for tool_use blocks:
`[Tool Call: ${block.name}](input: ${JSON.stringify(block.input)})`
// for tool_result blocks:
`[Tool Result: ${block.content?.map(c => c.text || c.title).join(', ')}]`
```

**Review by EurFelux — COMMENTED** [review]
## Review Summary

Thanks for the contribution! The implementation follows existing importer conventions well and the code is clean and well-structured.

### Issues Found

| Priority | Issue | Location |
|----------|-------|----------|
| 🔴 Critical | Model hardcoded to `claude-sonnet-4-6` — contradicts PR description's claim of dynamic mapping | `AnthropicImporter.ts:210-220` |
| 🟡 Moderate | `tool_use`/`tool_result` blocks silently dropped — tool-heavy conversations may lose content | `AnthropicImporter.ts:170-186` |
| 🟡 Moderate | Error swallowed without logging in catch block | `AnthropicImportPopup.tsx:53-55` |
| 🔴 Critical | **No unit tests** — project guidelines state "Features without tests are not considered complete". The 282-line importer needs test coverage for `validate()`, `parse()`, and edge cases (empty conversations, missing fields, tool-only messages) | — |

### What Looks Good

- Clean separation between parser and UI
- Proper i18n coverage (en/zh-CN/zh-TW)
- Good format detection with `!('mapping' in conv)` to distinguish from ChatGPT exports
- Follows existing `ChatGPTImporter` patterns consistently
- Graceful degradation on per-conversation failures

### Recommendation

Please add unit tests and address the model mapping issue (either implement dynamic mapping or update the PR description) before merging.

File: src/renderer/src/services/import/importers/index.ts [hunk 10]
```diff
@@ -1,12 +1,13 @@
+import { AnthropicImporter } from './AnthropicImporter'
 import { ChatGPTImporter } from './ChatGPTImporter'
 
 /**
  * Export all available importers
  */
-export { ChatGPTImporter }
+export { AnthropicImporter, ChatGPTImporter }
```
Comment (EurFelux) [inline src/renderer/src/services/import/importers/index.ts]: > [!NOTE]
> This issue/comment/review was translated by Claude.

Although exported, it is not actually used. Based on usage, the singleton from `availableImporters` should be used instead, so there's no need to export the class.

---
<details>
<summary>Original Content</summary>

虽然导出，却未实际使用。从用法来看，应该是使用`availableImporters`中的单实例，不需要再把类导出了。

</details>

File: src/renderer/src/services/import/importers/AnthropicImporter.ts [hunk 11]
```diff
@@ -0,0 +1,407 @@
...
+export class AnthropicImporter implements ConversationImporter {
+  readonly name = 'Claude'
+  readonly emoji = '🍒'
+
+  /**
+   * Validate if the file content is a valid Anthropic Claude export
+   */
+  validate(fileContent: string): boolean {
+    try {
+      const parsed = JSON.parse(fileContent)
```
Comment (DeJeune) [inline src/renderer/src/services/import/importers/AnthropicImporter.ts:116]: use zod

**Review by DeJeune — CHANGES_REQUESTED** [review]

**Comment by Pleasurecruise** [comment]
Superseded by #16164, which re-implements this feature on top of the current `main` (the v2 refactor made this branch too conflict-heavy to rebase) and folds in the code-review feedback from here. Closing in favor of #16164.
